html/semantics/scripting-1/the-template-element/template-element/template-content-node-document-document-properties.sub.html

```diff
@@ -0,0 +1,41 @@
+<meta charset=windows-1252>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+  test(() => {
+    assert_equals(document.origin, "{{location[scheme]}}://{{location[host]}}")
+    var t = document.createElement("template")
+    assert_equals(t.content.ownerDocument.origin, "{{location[scheme]}}://{{location[host]}}")
+  }, "document.origin")
+
+  test(() => {
+    assert_equals(document.domain, "www.web-platform.test")
```

Comment:
Create an iframe that loads a file with some subdomain using `{{domain[www]}}`, and then chop off the subdomain with e.g. `substring`+`indexOf` when setting `document.domain`. 

I *think* this would work for w3c-test.org and for browsers that use wptserve to run the tests. It still wouldn't work where wptserve is not used, but not much to do about that here.

But if setting should just throw then the test could instead just do `document.domain = document.domain` and assert that it throws, without iframes.